dexter/src/main/java/com/karumi/dexter/IntentProvider.java

```diff
@@ -0,0 +1,14 @@
+/*
+ * Copyright (C) 2015 Karumi.
+ */
```

Comment:
I've reviewed all the other files, it was missing in a bunch more but I've fixed them